src/query.ts

```diff
@@ -507,6 +510,45 @@ export class Query<T extends Model> {
     return _.pick(instance, Object.keys(getModelAssociations(this.model))) as T;
   }
 
+/**
+ * Returns the associations that correlate to the current include options
+ * @param {boolean} required? Whether or not to only get required associations (associations that are not nullable).
+ * @return {any[]} An array of association descriptors.
+ */
+  private getIncludedAssociations(required?: boolean): any[] {
```

Comment:
* `required` doesn't seem to be used
* Is it possible to make the `any[]` return type stricter here. Is there a Sequelize type you can use?